### ISSUE TITLE
Installing TensorFlow from wheel fails

### ISSUE THREAD

**Issue by spott** [kennethreitz/pipenv#960]
I'm attempting to install TensorFlow from a wheel, and I'm getting this:

```
❯ pipenv install --verbose pip/tensorflow_pkg/tensorflow-1.4.0rc1-cp36-cp36m-linux_x86_64.whl
Installing pip/tensorflow_pkg/tensorflow-1.4.0rc1-cp36-cp36m-linux_x86_64.whl…
⠋Installing 'pip/tensorflow_pkg/tensorflow-1.4.0rc1-cp36-cp36m-linux_x86_64.whl'
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/dist-packages/pkg_resources/_vendor/packaging/requirements.py", line 90, in __init__
    req = REQUIREMENT.parseString(requirement_string)
  File "/usr/local/lib/python3.6/dist-packages/pkg_resources/_vendor/pyparsing.py", line 1617, in parseString
    raise exc
  File "/usr/local/lib/python3.6/dist-packages/pkg_resources/_vendor/pyparsing.py", line 1607, in parseString
    loc, tokens = self._parse( instring, 0 )
  File "/usr/local/lib/python3.6/dist-packages/pkg_resources/_vendor/pyparsing.py", line 1379, in _parseNoCache
    loc,tokens = self.parseImpl( instring, preloc, doActions )
  File "/usr/local/lib/python3.6/dist-packages/pkg_resources/_vendor/pyparsing.py", line 3376, in parseImpl
    loc, exprtokens = e._parse( instring, loc, doActions )
  File "/usr/local/lib/python3.6/dist-packages/pkg_resources/_vendor/pyparsing.py", line 1383, in _parseNoCache
    loc,tokens = self.parseImpl( instring, preloc, doActions )
  File "/usr/local/lib/python3.6/dist-packages/pkg_resources/_vendor/pyparsing.py", line 3164, in parseImpl
    raise ParseException(instring, loc, self.errmsg, self)
pkg_resources._vendor.pyparsing.ParseException: Expected stringEnd (at char 3), (line:1, col:4)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.6/dist-packages/pkg_resources/__init__.py", line 2997, in __init__
    super(Requirement, self).__init__(requirement_string)
  File "/usr/local/lib/python3.6/dist-packages/pkg_resources/_vendor/packaging/requirements.py", line 94, in __init__
    requirement_string[e.loc:e.loc + 8]))
pkg_resources.extern.packaging.requirements.InvalidRequirement: Invalid requirement, parse error at "'/tensorf'"

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/bin/pipenv", line 11, in <module>
    sys.exit(cli())
  File "/usr/local/lib/python3.6/dist-packages/pipenv/vendor/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python3.6/dist-packages/pipenv/vendor/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python3.6/dist-packages/pipenv/vendor/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python3.6/dist-packages/pipenv/vendor/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python3.6/dist-packages/pipenv/vendor/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/lib/python3.6/dist-packages/pipenv/cli.py", line 1793, in install
    c = pip_install(package_name, ignore_hashes=True, allow_global=system, no_deps=False, verbose=verbose, pre=pre)
  File "/usr/local/lib/python3.6/dist-packages/pipenv/cli.py", line 1316, in pip_install
    if [x for x in requirements.parse(package_name.split('--hash')[0].split('--trusted-host')[0])][0].vcs:
  File "/usr/local/lib/python3.6/dist-packages/pipenv/cli.py", line 1316, in <listcomp>
    if [x for x in requirements.parse(package_name.split('--hash')[0].split('--trusted-host')[0])][0].vcs:
  File "/usr/local/lib/python3.6/dist-packages/pipenv/vendor/requirements/parser.py", line 50, in parse
    yield Requirement.parse(line)
  File "/usr/local/lib/python3.6/dist-packages/pipenv/vendor/requirements/requirement.py", line 220, in parse
    return cls.parse_line(line)
  File "/usr/local/lib/python3.6/dist-packages/pipenv/vendor/requirements/requirement.py", line 198, in parse_line
    pkg_req = Req.parse(line)
  File "/usr/local/lib/python3.6/dist-packages/pkg_resources/__init__.py", line 3042, in parse
    req, = parse_requirements(s)
  File "/usr/local/lib/python3.6/dist-packages/pkg_resources/__init__.py", line 2990, in parse_requirements
    yield Requirement(line)
  File "/usr/local/lib/python3.6/dist-packages/pkg_resources/__init__.py", line 2999, in __init__
    raise RequirementParseError(str(e))
pkg_resources.RequirementParseError: Invalid requirement, parse error at "'/tensorf'"
```

doing a `pipenv shell` and then `pip install pip/tensorflow_pkg/tensorflow-1.4.0rc1-cp36-cp36m-linux_x86_64.whl` seems to work.

##### Describe you environment

1. OS Type - Ubuntu 17.10

1. Python version: Python 3.6.3
1. Pipenv version: pipenv, version 8.2.7

##### Expected result

The install of TensorFlow from the wheel.

##### Actual result

see above.

**Comment by nateprewitt** [comment]
Hey @spott, this looks to be a duplicate of #870 so I'm closing this in favor of tracking the issue there. Thanks for the report!

**Comment by techalchemy** [comment]
Also duplicated in #936 #939 #949 #817 #540 #914 and should be resolved with PR #958 when I finish putting it all together

**Comment by techalchemy** [comment]
This is now resolved in #958
